src/stryker/quickstart.pug

```diff
@@ -18,52 +18,59 @@ block content
     section.row
```

Comment:
Is the `stryker-install.gif` above here still up to date?

Reply:
No, but it's so fast that people probably can't see it anyways 🚀

Reply:
Not 100% up-to-date but close enough EMO